src/phoneField/index.tsx

```diff
@@ -98,7 +100,7 @@ const iso2toDialCode = (iso2Value: string) => {
   }
   const countryPhoneData = findIso2(iso2Value)
   if (countryPhoneData) {
-    return `+${countryPhoneData.dialCode}`
+    return countryPhoneData.dialCode
```

Comment:
What is this update related to? It is not reflected in the changelog

Reply:
I added the "+" directly into `formattedCountriesPhoneData`, avoiding re-adding it on many place of the code.
I'll add it into the changelog.

Reply:
@loick If it's an internal change, it shouldn't be in the Changelog